utils/createDummyData.ts

```diff
@@ -1,71 +1,68 @@
 import { dbConnection, disconnectDB } from '../src/db/DBConnection';
-
 import { ArticleModel } from '../src/entities/Article';
 
 (async () => {
   await dbConnection();
   const articles = [
     {
       title: 'orko and tedi backend ',
-      publication: 'dailysun',
+      publicationID: '5fac75af7f68e1f1a4ae8b1b',
       artilceURL: 'www.orko.com',
       imageURL: 'www.orkopics.com',
       date: new Date('10-14-2020'),
-      likes: 0,
+      shoutouts: 0,
     },
     {
       title: 'appdev router sucks ---> let me tell you why',
-      publication: 'nooz',
+      publicationID: '5fac75af7f68e1f1a4ae8b1b',
       articleURL: 'www.jack.com',
       imageURL: 'www.jack.com',
       date: new Date('10-15-2020'),
-      likes: 0,
+      shoutouts: 0,
     },
     {
       title: 'Cooking w Cornell Hotel School',
-      publication: 'creme',
-      articleURL: 'www.ne',
-      imageURL: 'www.pic',
+      publicationID: '5fac75af7f68e1f1a4ae8b1b',
+      articleURL: 'www.new',
+      imageURL: 'www.pic.com',
       date: new Date('10-16-2020'),
-      likes: 5,
+      shoutouts: 5,
     },
     {
       title: 'Conners 3 hats - what each of them mean',
-      publication: 'slope',
-      articleURL: 'www.ne',
-      imageURL: 'www.pic',
+      publicationID: '5fac75af7f68e1f1a4ae8b1b',
+      articleURL: 'www.new.com',
+      imageURL: 'www.pic.com',
       date: new Date('10-17-2020'),
-      likes: 0,
+      shoutouts: 0,
     },
     {
       title: 'We need better articles',
-      publication: 'advocate',
-      articleURL: 'www.ne',
-      imageURL: 'www.pic',
+      publicationID: '5fac75af7f68e1f1a4ae8b1b',
+      articleURL: 'www.newhat.com',
+      imageURL: 'www.pic.com',
       date: new Date('10-21-2020'),
-      likes: 0,
+      shoutouts: 0,
```

Comment:
does setting all of these to the same `publicationID` not produce a crash? We are enforcing uniqueness so this should be a problem? It might be better to just generate new dummy uuids on the fly via the uuid function (whatever it is)

Reply:
To address it crashing, I could be wrong but I think uniqueness is enforced upon entry into the database but in testing here, I've just creating Article typed objects and comparing equality of the objects then testing the compareTrendiness query. But if I 1. create field resolver for trendiness and 2. add a function to make the objects on the fly this shouldn't be a problem anyways.

Reply:
Oh, I just realized all this means is that they're all from the articles from the same publication which is actually fine, doesn't affect the test, and is a realistic result.